VTEX - VTEX Do API.json

```diff
@@ -141,31 +232,37 @@
                         "explode": true,
                         "schema": {
                             "type": "integer",
-                            "description": "Number of notes per page. Maximum: 30.",
                             "example": 20
                         }
                     },
                     {
                         "name": "page",
                         "in": "query",
-                        "description": "Number of the page to be retrieved.",
+                        "description": "Number of pages to be retrieved.",
                         "required": false,
                         "style": "form",
                         "explode": true,
                         "schema": {
                             "type": "integer",
-                            "description": "Number of the page to be retrieved.",
                             "example": 3
                         }
                     },
                     {
-                        "$ref": "#/components/parameters/Content-Type"
+                        "name": "reason",
+                        "in": "query",
+                        "description": "This parameter is relevant only for PII-compliant accounts. When sending requests to this endpoint, PII-compliant accounts can use this parameter to declare the reason for requesting unmasked data. Otherwise, this endpoint will return masked PII data.",
```

Comment:
The text was already like this in the documentation, but before the field was declared in "components". Since it appeared only twice, I removed the mention from "components" to the usual parameters in "path". I did it thinking about not overusing "$ref" and having a more organized code.

After your comment, I checked it with Julia and she suggested changing it, so I did, and I agree it is better!